packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -462,6 +500,14 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
       onStartLocalVideo,
       onDisposeRemoteStreamView,
       onDisposeLocalStreamView,
+      /* @conditional-compile-remove(raise-hands) */
+      onRaiseHand,
+      /* @conditional-compile-remove(raise-hands) */
+      onLowerHand,
+      /* @conditional-compile-remove(raise-hands) */
+      onLowerHands,
+      /* @conditional-compile-remove(raise-hands) */
```

Comment:
Again, either onToggle or (on/off) style is enough, duplicated api is not needed